odoo/addons/base/models/ir_ui_view.py

```diff
@@ -564,7 +564,7 @@ def create(self, vals_list):
                         values['arch_updated'] = False
             values.update(self._compute_defaults(values))
 
-        self.clear_caches()
+        self.env.registry.clear_cache()
```

Comment:
Related to the three comments of this files, I forgot to add the template cache. It is now done and this will invalidate the template cache.

Assets do no depends on template cache. An assets is defined by ir.assets, attachments, manifest content and static files, nor views.